docs/monitoring/02-01-monitoring.md

```diff
@@ -2,32 +2,46 @@
+- **Grafana** that provides a dashboard and graph editor to visualize metrics collected from the Prometheus API. Grafana uses the query language called [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/) to select and aggregate the metrics data from the Prometheus database. To access the Grafana UI, use the `https://grafana.{DOMAIN}` address, where {DOMAIN} is the domain of your Kyma cluster.
 
-## Monitored Data sources
+  For more details, [read](https://grafana.com/docs/guides/getting_started/) the official documentation.
 
-Prometheus contains the flexibility to poll data from a variety of sources. Virtual machines on which Kubernetes runs make time-stamped data available, reporting on jobs started, workload, CPU performance, capacity, and more. In this case, the Service Monitor watches the Kubernetes API master to detect any job creation. The job produces time-stamped data that Prometheus consumes.
+- **Alertmanager** that receives alerts from Prometheus and forwards this data to configured Slack or Victor Ops channels.
 
-Pods may contain applications with custom metrics that Prometheus can poll through the Prometheus exporter.
+  > **NOTE:** There are no notification channels configured in the default monitoring installation. The current configuration allows you to add either Slack or Victor Ops channels.
 
-## Grafana
+  For more details, [read](https://prometheus.io/docs/alerting/alertmanager/) the official documentation.
 
-Kyma employs Grafana as a third-party resource in `kube-prometheus` to deliver a feature-rich metrics dashboard and graph editor.
+## Related resources
 
-To access the Grafana UI, use the following URL: `https://grafana.{DOMAIN}`. Replace DOMAIN with the domain of your Kyma cluster.
+Monitoring in Kyma also relies heavily on these custom resources:
 
-## Alertmanager
+- **Alert rules** define alert conditions for metrics. They are configured in Prometheus as PrometheusRule custom resource definitions (CRDs). Kyma provides a set of out-of-the-box alert rules that are passed from Prometheus to Alertmanager. The definitions of such rules specify the alert logic, the value at which alerts are triggered, the alerts' severity, and more. If you pre-define specific Slack or Victor Ops channels, Alertmanager fires the alerts onto the channel each time the alerts are triggered.
 
-Alertmanager receives harvested metrics from Prometheus and forwards this data on to the configured channels, such as email or incident management systems.
+- **Service Monitors** are custom resource definitions that specify the endpoints from which Prometheus should poll the metrics. Even if you expose a handful of metrics in your application, Prometheus polls only those from the endpoints specified in a Service Monitor custom resource definitions.
```

Comment:
```suggestion
- **Service Monitors** are custom resource definitions that specify the endpoints from which Prometheus should poll the metrics. Even if you expose a handful of metrics in your application, Prometheus polls only those from the endpoints specified in the Service Monitor custom resource definitions.
```
In the previous section you introduced the CRD abbreviation, should we use it here? (`custom resource definitions` shows up  twice in the extract).